Draganddroptest

gshields5541 opened the pull request:
Created branch draganddroptest for consideration to merge with development branch. Work Time: 2 hours.

gshields5541 committed:
This is the first commit. It's an image uploader that allow user to upload single or multiple images file by drag and drop or select file dialog.

The selected images are uploaded and displayed accordingly. The uploader only allow images.

gshields5541 committed:
This is the first commit.

JLaveta commented:
@gshields5541 Hey, so after I approved this merge I realized that it looks like you may have setup a project within the project. The "file-upload-vue" directory looks like it contains an entirely new Vue.js app. Can you work on fixing this? I'm a little confused on what is happening here, but I think we should only have one "src" directory, one "App.vue", one .gitignore, etc.

For now I have reverted the development branch back to the way ti was before the merge. Please correct and resubmit a pull request.

gshields5541 commented:
I’ve removed the folder that had the extra Vue.app and retested the correct one within src. Seems like it’s okay.

Gerald

> On May 16, 2019, at 1:34 AM, John Laveta <notifications@github.com> wrote:
> 
> @gshields5541 <https://github.com/gshields5541> Hey, so after I approved this merge I realized that it looks like you may have setup a project within the project. The "file-upload-vue" directory looks like it contains an entirely new Vue.js app. Can you work on fixing this? I'm a little confused on what is happening here, but I think we should only have one "src" directory, one "App.vue", one .gitignore, etc.
> 
> For now I have reverted the development branch back to the way ti was before the merge. Please correct and resubmit a pull request.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/ad440-cloud-project/UX-UI/pull/4?email_source=notifications&email_token=AE2C4EKZFAALHFNR27HQXNLPVUMALA5CNFSM4HNJK5UKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODVRCZRI#issuecomment-492973253>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AE2C4EPXOOPCVTP3IG3VTTTPVUMALANCNFSM4HNJK5UA>.
>
